examples/scenes/src/svg.rs

```diff
@@ -71,6 +71,38 @@ fn example_scene_of(file: PathBuf) -> ExampleScene {
     }
 }
 
+fn render_svg_rec(items: &[crate::pico_svg::Item]) -> Scene {
+    let mut scene = Scene::new();
```

Comment:
It's probably premature optimisation to re-use a single child scene for the top-level of each child, to cut down on the number of allocations slightly.